bindings/experimental/distrdf/tests/backend/test_common.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+
```

Comment:
Could you call the test directory `test` instead of `tests`? Just for consistency with PyROOT and cppyy